app/models/requirement.rb

```diff
@@ -30,6 +30,13 @@ def assignees
     assignments.active.map { |a| a.person }
   end
 
+  def assignable?(person)
+    return false if number_filled >= maximum_people
+    return false if people.include? person
```

Comment:
This is OK for now, but won't work long term. Requirements may span days, and a person may fulfill the same requirement on different days.

Reply:
@masukomi Would you have time to address this prior to bugging out ?